src/wp-admin/includes/update-core.php

```diff
@@ -1548,6 +1537,101 @@ function _copy_dir( $from, $to, $skip_list = array() ) {
 	return true;
 }
 
+/**
+ * Removes files from the `$_old_files` array after a core update.
+ *
+ * @ignore
+ * @since 6.1.0
+ *
+ * @see update_core()
+ *
+ * @global array $_old_files
+ *
+ * @param array $checksums An array of checksums for the new WordPress version.
+ */
+function _cleanup_old_files( $checksums ) {
+	global $_old_files;
```

Comment:
need to change this to `global $wp_filesystem, $_old_files` and add ` * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.` to the DocBlock

Reply:
Nicely spotted @pbiron!